app/src/androidTest/java/com/lastaoutdoor/lasta/ui/MainActivityTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.lastaoutdoor.lasta.ui
+
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@HiltAndroidTest
+class MainActivityTest {
+
+  @get:Rule(order = 0) val hiltRule = HiltAndroidRule(this)
+
+  @Test
+  fun testMainActivityIsLaunched() {
```

Comment:
It would be nice to rename this test `testMainActivityIsLaunchedAndLanguageIsUpdated()` and just add a modifier on some language component and test it is called or something like that if possible !